apps/extension/src/constants/content-script.ts

```diff
@@ -0,0 +1,3 @@
+export const CONTENT_SCRIPT = 'oko-contentscript';
+export const INPAGE = 'oko-inpage';
+export const PROVIDER = 'oko-provider';
```

Comment:
why do you have 2 different variables with the same value?
`libs/ui/src/utils/dapp.utils.ts`